### ISSUE TITLE
Fix ISRCTN identifiers we got from EUCTR

### ISSUE THREAD

**Issue by vitorbaptista** [opentrials/opentrials#430]
The identifiers should be in the form `ISRCTN09988575`, but they're quite messy. Check the `isrctn_international_standard_randomised_controlled_trial_numbe` column in `euctr` table in the warehouse.

**Comment by pwalsh** [comment]
@vitorbaptista is this issue covered by your work on identifiers? If yes, please close this one.

**Comment by vitorbaptista** [comment]
@pwalsh No, this isn't related to the identifiers refactoring, so still relevant
